tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas>=1.0,<1.1
+pytest==6.2.1
+allure-pytest==2.8.29
```

Comment:
Same question, shouldn't you put all that in a file specific to your lib in tests?

Reply:
For me, these packages are generic, so we could use it for all libs

Reply:
@HenriChabert is that OK with you? Should we resolve?

Reply:
Makes sense, resolved.